src/index.js

```diff
@@ -1,6 +1,10 @@
-var _       = require("lodash"),
-    assert  = require("assert-plus"),
-    Promise = require("bluebird");
+var assign   = require("lodash.assign"),
+    concat   = require("lodash.concat"),
+    forEach  = require("lodash.foreach"),
+    get      = require("lodash.get"),
+    toArray  = require("lodash.toarray"),
+    assert   = require("assert-plus"),
+    Promise  = require("bluebird");
```

Comment:
Sorry -- I meant all of these should be alphabetized, not just within the lodash modules. So it should be:

```
var assert  = require("assert-plus"),
    assign  = require("lodash.assign"),
    concat  = require("lodash.concat"),
    forEach = require("lodash.foreach"),
    get     = require("lodash.get"),
    Promise = require("bluebird"),
    toArray = require("lodash.toarray");
```

Reply:
Oops! I sorted them before removing the lodash (__) prefix and hence the weird sort order.